internal/service/alarms/internal/db/repo/alarms_repository.go

```diff
@@ -190,33 +187,42 @@ func getGetAlertNameAndResourceTypeID(am *api.AlertmanagerNotification) []bob.Ex
 
 // ArchiveResolvedAlarmEventRecord Delete resolved and move to archive
 func (ar *AlarmsRepository) ArchiveResolvedAlarmEventRecord(ctx context.Context) error {
```

Comment:
Might want to pluralize the function name since it is handling multiple records.

Reply:
Updated